.env

```diff
@@ -4,7 +4,7 @@ CELERY_RESULT_BACKEND=redis://redis
 API_URL=http://api
 DB_HOST=db
 DB_PORT=5432
-DB_USER=test
+DB_USER=api
```

Comment:
I would use another name, as this is not the user used by postgrest

Reply:
I don't understand what you mean by that?

Reply:
This is confusing with the role used by PostGREST here
https://github.com/enermaps/enermaps/blob/378e04420e81317397e7fae3cfbbfa8cf8177db3/db/init/postgrest.sql#L6

Reply:
1. PostgREST is still not on master. This was just to comply with the current setup.
2. The username here could be whatever is preferred. Ok so I'll set it back to the previous user name "test"

Reply:
Done. pushed.